### DISCUSSION TITLE
Отображение таблицы-дерева с агрегацией по дням

### DISCUSSION THREAD

**Discussion by erpic-ru** [totumonline/totum-ru-issues-and-discussions#304] [Как это сделать?] [answered]
Можно ли в текущих возможностях дерева в Тотуме сделать что-то похожее из временной таблицы, т.е. агрегировать табличную часть по дням?

<img width="745" alt="Снимок экрана 2023-01-07 в 11 31 18" src="https://user-images.githubusercontent.com/91674114/211141679-d008b499-8d29-4273-a038-5a17fa2edeb5.png">

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Дерево можно посчитать заранее и хранить где-то в поле.

Вот здесь это описано: https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/293

Потом мы смотрим, какие у нас даты есть в интервале, который мы хотим показать и делаем дереву roots по этим датам.

Тк дерево загружается всегда целиком то лучше каждый год держать в отдельной строке и загружать из нужной.
